src/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Tests
 {
+    [Collection("CultureTestCollection")]
```

Comment:
I'd rather turn off xunit parallelism altogether than have RemoteExecutor become the go-to fix for every shared state problem. Introducing RemoteExecutor into a test project is **very** costly for ILC runs, let alone the added hassles in not being able step through the remote delegate in the debugger.

I was (and still am) tempted to split off the small handful of RemoteExecute tests in System.Runtime.Tests into their own project so that someone investigating one of the other 99% tests on ILC don't have to pay the RemoteExecutor tax.